charts/bitwardenrs/1.1.1/values.yaml

```diff
@@ -0,0 +1,178 @@
+#     type: ""
+#     # Database host. Required if external database is enabled.
+#     host: ""
+#     # Database port. Optional, default value is specific to the database backend.
+#     port: ""
+#     # Database name.
+#     database: ""
+#     # Database user.
+#     user: ""
+#     # Database password. Special characters must be escaped with percent encoding.
+#     password: ""
+#     # Use existing secret for database credentials.
+#     existingSecret:
+#       enabled: false
+#       name: ""
+#       userKey: ""
+#       # Special characters in the password value must be escaped with percent encoding.
+#       passwordKey: ""
+#   # Enable SMTP. https://github.com/dani-garcia/bitwarden_rs/wiki/SMTP-configuration
+#   smtp:
+#     enabled: false
+#     # SMTP hostname, required if SMTP is enabled
+#     host: ""
+#     # SMTP sender e-mail address, required if SMTP is enabled
+#     from: ""
+#     # SMTP sender name, defaults to 'Bitwarden_RS'
+#     fromName: ""
+#     # Enable SSL connection
+#     ssl: true
+#     # SMTP port
+#     port: 587
+#     # SMTP username
+#     user: ""
+#     # SMTP password. Required is user is specified, ignored if no user provided
+#     password: ""
+#     # Use existing secret for SMTP authentication
+#     existingSecret:
+#       enabled: false
+#       name: ""
+#       userKey: ""
+#       passwordKey: ""
+#   # Enable Yubikey 2FA: https://github.com/dani-garcia/bitwarden_rs/wiki/Enabling-Yubikey-OTP-authentication
+#   yubico:
+#     enabled: false
+#     # OTP verification server. Will use the default YubiCloud servers if not specified
+#     server: ""
+#     # API Client ID for OTP server. Ignored if existingSecret is provided.
+#     clientId: ""
+#     # API Secret Key for OTP server. Required if clientId is specified, ignored when using existingSecret.
+#     secretKey: ""
+#     # Use existing secret for API keys
+#     existingSecret:
+#       enabled: false
+#       name: ""
+#       clientIdKey: ""
+#       secretKeyKey: ""
+
+# env: {}
+# # If you plan to run the WebUI on a port other than port 80, specify that here:
+# # For example, if running the container as a non-root user.
+# #  ROCKET_PORT: "80"
+
+# persistence:
+#   type: statefulset
+#   enabled: false
+#   size: 1Gi
+#   accessMode: ReadWriteOnce
+#   ## Persistent Volume storage class
+#   # storageClass: "-"
+#   ## Use existing Persistent Volume Claim
+#   # existingClaim:
+
+# serviceAccount:
+#   # Specifies whether a service account should be created
+#   create: true
+#   # Annotations to add to the service account
+#   annotations: {}
+#   # The name of the service account to use.
+#   # If not set and create is true, a name is generated using the fullname template
+#   name: ""
+
+# podAnnotations: {}
+
+# podSecurityContext: {}
+#   # fsGroup: 2000
+
+# securityContext: {}
+#   # capabilities:
+#   #   drop:
+#   #   - ALL
+#   # readOnlyRootFilesystem: true
+#   # runAsNonRoot: true
+#   # runAsUser: 1000
```

Comment:
Currently we do not actively support these settings, but its on the radar, because they are rather important for setting up secure containers. Setting it up in a non-breaking way, thats not going to affect the ease-of-install needs some carefull consideration though.